### DISCUSSION TITLE
[Documentation, Screenshots] Would you folks consider adding a few screenshots/images of the main UI/usage pattern of solidus?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [solidusio/solidus#4936] [New Features or Ideas]
Hey there,

I just "randomly" found the project by following someone else who is more into rails.

I read the main README but I am not entirely certain what solidus really is, despite the words describing it. :)

Would you guys possibly considering adding a few screenshots? This does not have to be extensive, just,
say, about 3 images in total that should give a first impression. That way we "downstream" folks can
have a better idea what solidus is about. Thank you for reading this issue request - please feel free to
close it at any moment in time.

**Comment by waiting-for-dev** [comment]
Hey @rubyFeedback, we have https://demo.solidus.io/, linked from the README, for a visual demo. Does that work for you?
